packages/@react-spectrum/searchwithin/src/SearchWithin.tsx

```diff
@@ -60,20 +76,34 @@ function SearchWithin(props: SpectrumSearchWithinProps, ref: FocusableRef<HTMLEl
     isRequired,
     label: null,
     isQuiet: false,
-    'aria-labelledby': labelProps.id || ariaLabel,
-    validationState: null
+    validationState: null,
+    description: null,
+    errorMessage: null,
+    descriptionProps: null,
+    errorMessageProps: null
   };
+
   let searchFieldClassName = classNames(styles, 'spectrum-SearchWithin-searchfield');
   let pickerClassName = classNames(styles, 'spectrum-SearchWithin-picker');
+
   let slots = {
-    searchfield: {UNSAFE_className: searchFieldClassName, ...fieldProps, ...defaultSlotValues},
-    picker: {UNSAFE_className: pickerClassName, menuWidth, align: 'end', ...defaultSlotValues}
+    searchfield: {
+      ...defaultSlotValues,
+      UNSAFE_className: searchFieldClassName,
+      // Apply aria-labelledby of group or the group id to searchfield. No need to pass the group id (we want a new one) and aria-label (aria-labelledby will suffice)
+      'aria-labelledby': labelledBy
+    },
+    picker: {
+      ...defaultSlotValues,
+      id: pickerId,
+      UNSAFE_className: pickerClassName,
+      menuWidth,
+      align: 'end',
+      'aria-label': formatMessage('searchWithin'),
```

Comment:
Think the intention here is that the user provides an `aria-label` for the component as a whole and that gets concatenated with the label for the individual parts of the SearchWithin element (picker + search field). Not sure we want to allow the user to specify the label for every sub component within the SearchWithin since they each serve a specific function

Reply:
Yep, I think having it defined is what we want. if this is something we need to change it could be a follow-up